package.json

```diff
@@ -47,6 +49,8 @@
     "ganache-cli": "6.1.8",
     "mocha": "5.2.0",
     "truffle": "5.0.6",
+    "ts-generator": "0.0.8",
+    "typechain": "^0.3.14",
```

Comment:
At least according to the _current version_ of the [dev guidelines], you should not use the caret operator here.

[dev guidelines]: https://github.com/OpenST/developer-guidelines/#managing-internal-and-external-dependencies